nmostesting/GenericTest.py

```diff
@@ -383,6 +397,97 @@ def validate_schema(self, payload, schema):
     def do_request(self, method, url, **kwargs):
         return TestHelper.do_request(method=method, url=url, **kwargs)
 
+    def make_key_cert_files(self, cert_file, key_file):
+        # create a key pair
+        k = crypto.PKey()
+        k.generate_key(crypto.TYPE_RSA, 2048)
+
+        # create cert
+        cert = crypto.X509()
+        cert.set_version(2)
+        cert.get_subject().C = "GB"
+        cert.get_subject().ST = "England"
+        cert.get_subject().O = "NMOS Testing Ltd"  # noqa: E741
+        ca_cert_subject = cert.get_subject()
+        ca_cert_subject.CN = "ca.testsuite.nmos.tv"
+        cert.set_issuer(ca_cert_subject)
+        cert.get_subject().CN = "mocks.testsuite.nmos.tv"
```

Comment:
Are these fixed, or should they be based on `CONFIG.DNS_DOMAIN`? I think we have some instructions somewhere that explain if you do want to use a different DNS domain, you also need to replace the CA root cert etc...

Reply:
Fixed